packages/components/pie-webc-core/src/mixins/formControl/formControlMixin.ts

```diff
@@ -50,6 +50,8 @@ export const FormControlMixin =
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             constructor (...args: any[]) {
                 super(...args);
+                // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
We've added these disable lines for now and will write a ticket to address this typescript errors.

<img width="1486" height="155" alt="Screenshot 2025-07-23 at 11 54 10" src="https://github.com/user-attachments/assets/a92000ba-057c-44ea-a36a-7e5021b9b098" />